src/middlewared/middlewared/plugins/service_/services/all.py

```diff
@@ -22,6 +23,8 @@
 
 from .pseudo.ad import ActiveDirectoryService, LdapService, NisService
 from .pseudo.collectd import CollectDService, RRDCacheDService
+from .pseudo.docker_linux import DockerService
```

Comment:
I've imagined pseudo services as ones that were really one-shot FreeBSD rc scripts and we'll eliminate them all someday. These two seem to be real services.